chat/src/main/kotlin/rys/ajaxpetproject/chat/application/service/MessageServiceKludge.kt

```diff
@@ -0,0 +1,63 @@
+package rys.ajaxpetproject.chat.application.service
+
+import org.springframework.stereotype.Service
+import reactor.core.publisher.Flux
+import reactor.core.publisher.Mono
+import rys.ajaxpetproject.chat.application.port.out.MessageServiceOutPort
+import rys.ajaxpetproject.chat.domain.Message
+import rys.ajaxpetproject.model.MongoMessage
+import rys.ajaxpetproject.service.MessageService
+
+@Service
+class MessageServiceKludge(
```

Comment:
> A "kludge" refers to a workaround or quick-and-dirty solution to a problem, typically inelegant but functional. It's often used in software development to describe a temporary or makeshift solution.

Reply:
Let me explain: 
That's not clean version. I wanted to say that in application we may have our own MessageService BUT I didn't want to create one. This really should be in Infrastructure layer

Reply:
Actually I'll move this class. Looks better in infrastructure